src/custom_request_schedule_manager.cc

```diff
@@ -52,62 +61,35 @@ CustomRequestScheduleManager::CustomRequestScheduleManager(
           params.batch_size, params.measurement_window_ms, params.max_trials,
           params.max_threads, params.num_of_sequences,
           params.shared_memory_type, params.output_shm_size,
-          params.serial_sequences, parser, factory, params.request_parameters)
+          params.serial_sequences, parser, factory, params.request_parameters),
+      warmup_request_count_(params.warmup_request_count),
+      benchmark_request_count_(params.request_count)
```

Comment:
These values are now passed to `CustomRequestScheduleManager` at its creation.

The `warmup_request_count_` value is needed [here](https://github.com/triton-inference-server/perf_analyzer/pull/366/files#diff-208ed87aa8e88c62da2501d92468c47876fd526bd164a71fd4d9d1b4536194dbR119) in order to know how many payloads to consider as part of the warmup phase.

The `benchmark_request_count_` value is loosely needed [here](https://github.com/triton-inference-server/perf_analyzer/pull/366/files#diff-208ed87aa8e88c62da2501d92468c47876fd526bd164a71fd4d9d1b4536194dbR113) for extra validation.